internal/browser/browser.go

```diff
@@ -117,7 +126,7 @@ func StartChromeDebugMode() bool {
 	case "windows":
 		cmd = exec.Command(getWindowsChromeLocation(), "--remote-debugging-port=9222", "--user-data-dir=remote-profile")
 	case "darwin":
-		cmd = exec.Command("/Applications/Google Chrome.app/Contents/MacOS/Google Chrome", "--remote-debugging-port=9222", "--user-data-dir=remote-profile")
+		cmd = exec.Command("open", "-a", "/Applications/Google \\Chrome.app", "--args", "--remote-debugging-port=9222", "--user-data-dir=remote-profile")
```

Comment:
We were originally using this however it doesn't look like open -a properly supports --args for Chrome. The move to launching dthe app directly looks to have solved it, if you're running OSX can you verify this (I had a tested verify this change a few hours ago and the new strategy was working for him)